include/avif/avif.h

```diff
@@ -523,20 +523,20 @@ typedef struct avifCodecConfigurationBox
 struct avifIO;
 
 // Destroy must completely destroy all child structures *and* free the avifIO object itself.
-// This function pointer is optional, however, if the reader isn't intended to be owned by
+// This function pointer is optional, however, if the avifIO object isn't intended to be owned by
 // a libavif encoder/decoder.
 typedef void (*avifIODestroyFunc)(struct avifIO * io);
 
-// This function should return a block of memory that *must* remain valid until another call to this
-// avifIO struct is used (reusing a read buffer is acceptable/expected).
+// This function should return a block of memory that *must* remain valid until another read call to
```

Comment:
This change assumes a write call (not implemented yet) won't modify the internal read buffer.